### PR TITLE
CATROID-744 - Change Login dialog options

### PR DESCRIPTION
**Pull request by atharva137**
**Description**
Added cancel button on login dialog options.

**Jira Ticket Reference** :  https://jira.catrob.at/browse/CATROID-744

Attached Screenshot for the same.
login_Dialog_screen

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by juliajulie95** [comment]
Please also push the changes you did for fixing the tests (:

**Comment by atharva137** [comment]
Hi julia Herold Ma'am  , Yes I have pushed the changes for all the Login Test cases .

**Comment by atharva137** [comment]
@juliajulie95 
Hello Julia Herold Ma'am, I'm attaching screenshot to my previous comment . Please find it
LoginScreen

**Comment by juliajulie95** [comment]
Thank you, I will review it roday

**Review by juliajulie95 — CHANGES_REQUESTED** [review]
Please fix the linter warnings: https://jenkins.catrob.at/job/Catroid/job/PR-3909/9/analysis/
If you need any help with this, message me please (:

**Comment by atharva137** [comment]
@juliajulie95 
Hi Julia Herold Ma'am,  I have fixed  the linter warnings. Please review . 
Thank and Regards,
Atharva

File: catroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -815,7 +815,7 @@
     <string name="sign_in_dialog_title">Sign in</string>
     <string name="username">Username</string>
     <string name="password">Password</string>
-    <string name="password_forgotten">Forgot your password?</string>
+    <string name="reset_password">Reset password?</string>
```
Comment (juliajulie95) [inline catroid/src/main/res/values/strings.xml:818]: Please delete the "?" afterwards

**Review by juliajulie95 — CHANGES_REQUESTED** [review]
Thank you for your work!
Please delete the "?" after "reset password", because it is not a question but an action.
Everything else looks good to me and I will approve the pull request after this change.

**Comment by atharva137** [comment]
Hi @juliajulie95  ma'am,  I have deleted the "?" after "reset password" . please review. Attaching screenshot for the same.  Thank You !! - Atharva.

ResetPassword

**Review by juliajulie95 — APPROVED** [review]

**Comment by juliajulie95** [comment]
Please push the ticket in jira dashboard to in code review (:

**Comment by atharva137** [comment]
Hi @juliajulie95  Ma' am , I am trying to change the status from " Under Development"  to "Code Review" , but not able to update it .Could you please help me in this .

Thanks and Regards,
Atharva

**Comment by wslany** [comment]
Thanks @atharva137 !

**Comment by wslany** [comment]
Awesome, the OutgoingNetworkCalls now also are green again!!!!
image

**Pull request merged**